docs/release-notes-7.1.0/master.adoc

```diff
@@ -0,0 +1,28 @@
+:mta:
+include::topics/templates/document-attributes.adoc[]
+:toc:
+:toclevels: 3
+:numbered:
+
+:imagesdir: topics/images
+:context: release-notes
+:release-notes:
+:_content-type: ASSEMBLY
+= Release Notes
+
+//Inclusive language statement
+include::topics/making-open-source-more-inclusive.adoc[]
+
+[id="release-notes-introduction"]
+== Introduction
+{ProductFullName} {DocInfoProductNumber} accelerates large-scale application modernization efforts across hybrid cloud environments on {ocp-first}. This solution provides insight throughout the adoption process, at both the portfolio and application levels: inventory, assess, analyze, and manage applications for faster migration to {ocp-short} through the user interface.
```

Comment:
Introduction is same as abstract, may be later we need to think something better for introduction

Reply:
Yup, just haven't thought of it yet, but can be worked on later